### PR TITLE
Add quick plot function

### PR DESCRIPTION
**Pull request by awirb**
* Add quick plot function to compare two datasets
* Add functions to handle files and directories and use this within the modules
* Add plot flag in general configuration file
* Add Outputs folder and xml files to gitignore in avaframe folder

### CODE REVIEW

File: avaframe/avaframeCfg.ini [hunk 1]
```diff
@@ -4,4 +4,12 @@
...
-avalancheDir = data/avaHockeySmoothChannel
+avalancheDir = data/avaHockeySmoothSmall
+
+
+[FLAGS]
+# True if plots shall be plotted to screen
+showPlot = False
+
+# True if comparison to reference shall be performed
+compRef = True
```
Comment (matthiasto) [inline avaframe/avaframeCfg.ini:15]: should this be here? We are the only ones who will compare to the reference no?

File: avaframe/in3Utils/fileHandlerUtils.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def makeADir(dirName, flagKeepDir=False):
+    """ Make directories """
+
+    # If directory already exists - Delete directory first is default
+    if os.path.isdir(dirName):
+        if flagKeepDir:
```
Comment (matthiasto) [inline avaframe/in3Utils/fileHandlerUtils.py:24]: so you keep it this way where the default is remove dir?

Comment (awirb) [reply avaframe/in3Utils/fileHandlerUtils.py:24]: What is your opinion, shall I put it the other way round and make a FlagRemDir?

File: avaframe/avaframeCfg.ini (hunk 1 above)
Comment (awirb) [reply avaframe/avaframeCfg.ini:15]: Ok fine I remove it

File: avaframe/in3Utils/fileHandlerUtils.py (hunk 2 above)
Comment (matthiasto) [reply avaframe/in3Utils/fileHandlerUtils.py:24]: okay lets keep it like this

Comment (awirb) [reply avaframe/in3Utils/fileHandlerUtils.py]: I think you are right, I changed it to flagRemDir

**Pull request merged**